### ISSUE TITLE
ReferenceError: setUserData is not defined

### ISSUE THREAD

**Issue by c-goettert** [Countly/countly-sdk-react-native-bridge#73]
After upgrading to the latest sdk-version (20.11.13), we get the following error when calling `serUserData` :
```
ExceptionsManager.js:179 Unhandled promise rejection ReferenceError: setUserData is not defined
    at _callee13$ (Countly.js:520:32)
    at tryCatch (runtime.js:63:40)
    at Generator.invoke [as _invoke] (runtime.js:293:22)
    at Generator.next (runtime.js:118:21)
    at tryCatch (runtime.js:63:40)
    at invoke (runtime.js:154:20)
    at runtime.js:189:11
    at new Promise (es6.promise.js:177:7)
    at callInvokeWithMethodAndArg (runtime.js:188:16)
    at AsyncIterator.enqueue [as _invoke] (runtime.js:211:13)
```
As a result, no user properties are sent to Countly. Presumably the problem was introduced with the changes in version 20.11.12?

**Comment by ijunaid** [comment]
@c-goettert 
Thank you for reporting this issue. We have fixed it and published a new release (20.11.14).

**Comment by c-goettert** [comment]
@ijunaid thanks for this super quick fix 👍
